.osparc/jupyter-math/metadata.yml

```diff
@@ -10,7 +10,7 @@ description:
   "
 key: simcore/services/dynamic/jupyter-math
 version: 2.0.5
-integration-version: 1.0.0
+integration-version: 2.0.0
```

Comment:
Q: why is this changed? can you explain the difference between 1.0.0 and 2.0.0? 
This version has implication on several things in simcore, including the computational sidecar. are there specifics on how to run these 2.0.0 services?

Reply:
With @mguidon we decided that the services ran via `dynamic-sidecar` should have their `integration-version` bumped to `2.0.0`. There is no logic inside simcore that actually uses this.